connector/signaltometricsconnector/internal/aggregator/aggregator.go

```diff
@@ -87,6 +89,28 @@ func (a *Aggregator[K]) Aggregate(
 				v, v,
 			)
 		}
+	case pmetric.MetricTypeGauge:
+		// aggregated gauges are currently supported when used with the ExtractGrokPatterns OTTL function, where the result is a pcommon.Map with named capture groups
+		raw, err := md.Gauge.Value.Eval(ctx, tCtx)
+		if err != nil {
+			return fmt.Errorf("failed to execute OTTL value for gauge: %w", err)
+		}
+		v, ok := raw.(pcommon.Map)
```

Comment:
To support other use cases for gauges, IMO, we should definitely support numerical types.